src/main.rs

```diff
@@ -209,7 +213,9 @@ fn get_languages(env_lang: Option<&str>, env_language: Option<&str>) -> Vec<Stri
     if env_lang.is_none() {
         return vec!["en".to_string()];
     }
-    let env_lang = env_lang.unwrap();
+
+    let env_lang = env_lang
+        .expect("Sorry It Shouldn't Be Happened We've checked and the env lang is not None!");
```

Comment:
I think the version with `unwrap` is better here. If we really want to get rid of unwrap, we should use a let-else statement (which was not stable when this code was written)